src/database/models/index.js

```diff
@@ -0,0 +1,33 @@
+
+import fs from 'fs';
+import path from 'path';
+import Sequelize from 'sequelize';
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+import configs from '../config';
```

Comment:
nice work here, but all `imports` should be at the top of the file, for easy refactoring